controllers/apartments.rb

```diff
@@ -8,7 +11,19 @@
   erb :"apartments/new"
 end
 
+post '/apartments' do
+  @apartment = Apartment.create(params[:apartment])
+  redirect :"apartments/#{@apartment.id}"
```

Comment:
remember the initial slash here. Also, redirects can take strings as arguments, so no need for the intial colon here.